web/src/components/common/layout/Header.tsx

```diff
@@ -81,20 +79,18 @@ interface INavProps {
 const Nav: React.FC<INavProps> = (props) => {
   const { isAuthenticated, handleDDMobile, isDDMobileOpen, dirs } = props;
 
-  const { ref, isComponentVisible, setIsComponentVisible } = useComponentVisible(false);
+  const history = useHistory();
 
-  const handleDDDesktop = () => {
-    setIsComponentVisible(!isComponentVisible);
+  const handleCreateDesktop = () => {
+    //  newer versions of react uses useNavigate instead of useHistory from 'react-router-dom'
```

Comment:
Leave a TODO and your name at this line so we can come back to address it later when you have more experience.